modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="oc-mirror-creating-image-set-config_{context}"]
+= Creating the image set configuration
+
+// TODO: Decide if this should be in a procedure step, or in its own module
+
+// TODO: Intro
+
+// Prior to running `oc mirror` / mirroring images using the `oc-mirror` plug-in, you must create the `ImageSetConfiguration` resource to define... TODO.
+
+// TODO Need more information on how this config works. Do you use the same exact config both when doing mirror2disk and disk2mirror?
+
+.Procedure
+
+// TODO: better sentence
+. Create an `ImageSetConfiguration` resource that specifies the necessary configuration details:
++
+.Example `ImageSetConfiguration` file
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha1
+kind: ImageSetConfiguration
+archiveSize: 4 <1>
+mirror:
+ ocp:
+   channels:
+     - name: stable-4.9 <2>
+ operators:
+   - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.9 <3>
+ helm:
+   repos:
+     - name: redhat-helm-charts <4>
+       url: https://raw.githubusercontent.com/redhat-developer/redhat-helm-charts/master <5>
+       charts:
+         - name: ibm-mongodb-enterprise-helm <6>
+           version: 0.2.0 <7>
+ additionalImages:
+   - name: registry.redhat.io/ubi8/ubi:latest <8>
+storageConfig:
+ registry:
+   imageURL: example.com/example/oc-mirror <9>
+   skipTLS: false <10>
+----
+<1> Maximum size, in GiB, of each file within the image set.
+<2> Channel to retrieve the {product-title} version from.
+<3> Operator catalog to retrieve the Operator version from.
+<4> Helm repository name.
+<5> URL to query for chart information.
+<6> Helm chart to add to the image set.
+<7> Optional: Chart version to add to the image set.
+<8> Optional: Tag of image to add to the image set.
+<9> Optional: Back-end location to save the image set metadata to.
```

Comment:
@afflom @jpower432 We (Alex and I) had talked about only supporting the stateful method in the product docs, which I think meant that the `storageConfig` section is required. Is that true, and if so, we should probably state that here somehow.